person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..2000)
+    @name = name
+    @age = age
+    @permission = parent_permission
+  end
+
+  attr_reader :id, :name, :age
+
+  def names(name)
+    @name = name
+  end
+
+  def ages(age)
+    @age = age
+  end
+
+  def can_use_services?
+    true if @permission || is_of_age
+  end
```

Comment:
- [ ] When I try to use this method it throws an error "undefined local variable or method `is_of_age' ". Kindly figure out what's the matter.